Quaver.Shared/Skinning/SkinKeys.cs

```diff
@@ -742,9 +742,13 @@ private void LoadLaneSpecificElements()
                     const int snapCount = 9;
 
                     var objects = LoadSpritesheet(SkinKeysFolder.HitObjects, "note-hitobject-sheet", false, snapCount, 1);
+                    var holdobjects = LoadSpritesheet(SkinKeysFolder.HitObjects, "note-holdobject-sheet", false, snapCount, 1);
+
+                    if (holdobjects.Count == 1)
+                        holdobjects=objects;
```

Comment:
```suggestion
                        holdobjects = objects;
```
Should be a space in between